src/NexusMods.App.UI/Pages/MyGames/MyGamesView.axaml

```diff
@@ -25,25 +25,38 @@
                     Spacing="24">
 
             <Border x:Name="AddGamesToGetStarted"
-                    Padding="24"
-                    BorderThickness="1"
+                    Padding="0"
+                    BorderThickness="0"
                     BorderBrush="{StaticResource StrokeTranslucentWeakBrush}"
                     CornerRadius="8">
 
                 <StackPanel Spacing="24">
-                    <TextBlock Text="Add games to get started"
-                               Theme="{StaticResource HeadingXSSemiTheme}"
-                               Foreground="{StaticResource NeutralStrongBrush}" />
-                    <TextBlock x:Name="NoGamesDetectedText"
-                        Text="No games detected"
-                        Theme="{StaticResource BodyMDNormalTheme}"
-                        Foreground="{StaticResource NeutralSubduedBrush}" />
+                    <StackPanel Spacing="8">
+                        <StackPanel Orientation="Horizontal" Spacing="8">
+                            <Border Width="48" Height="48">
+                                <icons:UnifiedIcon Size="48" Value="{x:Static icons:IconValues.PictogramGame3D}" />
+                            </Border>
+                            <TextBlock Text="My Games"
+                                       Theme="{StaticResource HeadingSMSemiTheme}"
+                                       Foreground="{StaticResource NeutralStrongBrush}"
+                                       VerticalAlignment="Center" />
+                        </StackPanel>
+
+                        <TextBlock x:Name="AddGamesToGetStartedText"
+                                   Text="Add games to get started"
+                                   Theme="{StaticResource BodyMDNormalTheme}"
+                                   Foreground="{StaticResource NeutralStrongBrush}" />
+                        <TextBlock x:Name="NoGamesDetectedText"
+                                   Text="No games detected"
+                                   Theme="{StaticResource BodyMDNormalTheme}"
+                                   Foreground="{StaticResource NeutralSubduedBrush}" />
+                    </StackPanel>
                     <ItemsControl
                         x:Name="DetectedGamesItemsControl">
                         <ItemsControl.ItemsPanel>
                             <ItemsPanelTemplate>
-                                <panels:FlexPanel 
-                                    RowSpacing="24" 
+                                <panels:FlexPanel
+                                    RowSpacing="24"
                                     ColumnSpacing="24"
                                     Wrap="Wrap" />
                             </ItemsPanelTemplate>
```

Comment:
This should be using `EmptyState`.

Reply:
Good point

Reply:
It's now using the `EmptyState` control and I've changed the code so that if Header or Icons aren't set, then those controls `IsVisibility =False` so that they aren't using up empty UI real estate. This matches other controls. Also removed some extra styles and needed to tweak some just for it's usage here.